src/jsdom.js

```diff
@@ -29,7 +29,8 @@ function fromSource(src, options) {
             FetchExternalResources: ['script'],
             ProcessExternalResources: ['script']
         },
-        virtualConsole: jsdom.createVirtualConsole().sendTo(console)
+        virtualConsole: jsdom.createVirtualConsole().sendTo(console),
+        created: options.jsdomCreated
```

Comment:
Sorry, couldn't find this briefly poking around the jsdom docs/code. Could you point me to the docs/code on this param and what it does?